### PR TITLE
[CIRCLE-38943] Rerun of related resource experiment with new position

### PR DESCRIPTION
**Pull request by ccgeller**
Rerun (current)Ticket: [CIRCLE-38943](https://circleci.atlassian.net/browse/CIRCLE-38943)
Original Ticket: [CIRCLE-37092](https://circleci.atlassian.net/browse/CIRCLE-37092)

# Description
- Updated experiment key
- Change position that `related resources` will appear

preview: [link](http://circleci-doc-preview.s3-website-us-east-1.amazonaws.com/CIRCLE-38943-article-suggester-preview/?force-all)

Note not all pages will have the change. See images below to see all pages effected.

# Reasons
We are going to run an experiment on the top 5 doc pages for an MVP version of the Article Suggester. This is a fairly simple experiment (nothing is smart/dynamic and selects the suggested articles) because we first want to validate that this is something users want. Pages + Links can be found here: https://docs.google.com/document/d/1Mg5kXWXfM2OstgqMH8ybN92KmTkqsDs-137NvuU-0gM/edit#

# Screenshots

in control:
Screen Shot 2021-09-20 at 12 24 33 PM

Previous treatment: 
Screen Shot 2021-09-25 at 3 00 21 PM

in treatment:
Screen Shot 2021-11-09 at 11 41 36 AM

# Details 

**Configuration Reference**
Screen Shot 2021-11-09 at 11 41 01 AM

**Env Vars**
Screen Shot 2021-11-09 at 11 42 05 AM

**Workflows**
Screen Shot 2021-11-09 at 11 42 22 AM

**Sample Config**
Screen Shot 2021-11-09 at 11 42 38 AM

**Local CLI**
Screen Shot 2021-11-09 at 11 42 55 AM

### CODE REVIEW

**Comment by teesloane** [comment]
I cloned your branch and I'm not seeing the related resources locally. Do you mind putting up a preview branch for testing there?

File: src/js/experiments/relatedResources.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 $(() => {
-  // https://app.optimizely.com/v2/projects/16812830475/experiments/20638820544/variations
+  // https://app.optimizely.com/v2/projects/16812830475/experiments/20872380274/variations
   window.OptimizelyClient.getVariationName({
-    experimentKey: 'dd_docs_related_resources_mvp_test',
-    groupExperimentName: 'q3_fy22_docs_disco_experiment_group_test',
+    experimentKey: 'dd_docs_related_resources_mvp_test_rerun',
```
Comment (tshen815) [inline src/js/experiments/relatedResources.js:4]:
```suggestion
    experimentKey: 'dd_docs_related_resources_mvp_test_rerun_test',
```

File: src/js/experiments/relatedResources.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+    groupExperimentName: 'q4_fy22_docs_disco_experiment_group_test',
```
Comment (tshen815) [inline src/js/experiments/relatedResources.js:5]: due to some inaccuracies for the experiment viewed data that gets collected, we have to add in a new option moving forward. All experiments now also need to have the `experimentContainer` key. You can look more into it here: https://github.com/circleci/circleci-docs/pull/5921

**Review by tshen815 — CHANGES_REQUESTED** [review]
Pulled locally and it looks good! Left a new comments that need to be addressed, mainly the new `experimentContainer` key that needs to be added for all experiments. Basically we have to have an html element now with either a class or id that only occurs for that experiment that wraps all the contents of the experiment. Romain had a PR for that feature that you can look into [here](https://github.com/circleci/circleci-docs/pull/5921)

File: src/js/experiments/relatedResources.js (hunk 2 above)
Comment (ccgeller) [reply src/js/experiments/relatedResources.js:5]: Good catch! Needed to rebase with master :)

**Review by tshen815 — APPROVED** [review]
Pulled and tested, looks good!

**Pull request merged**